README.md

```diff
@@ -122,7 +122,7 @@ in Unity Editor.
 
     namespace UIWidgetsSample
     {
-        public class CountDemo : UIWidgetsPanel
+        public class UIWidgetsExample : UIWidgetsPanel
```

Comment:
The class name has to match the script file name in order for the script to be attached to the canvas panel.